r2/r2/lib/captcha.py

```diff
@@ -48,10 +48,14 @@ def getLayers(self, solution="blah"):
                  Distortions.SineWarp()))
 
 def get_iden():
-    return randomIdentifier(length=IDEN_LENGTH)
+    return self.randomIdentifier(length=IDEN_LENGTH)
 
 def make_solution():
-    return randomIdentifier(alphabet=string.ascii_letters, length = SOL_LENGTH).upper()
+    return self.randomIdentifier(alphabet=string.ascii_letters, length = SOL_LENGTH).upper()
+
+def randomIdentifier(alphabet = string.ascii_letters + string.digits,
```

Comment:
Since you're defining your own randomIdentifier anyway, the import at the top is no longer needed